src/test/java/org/autopotato/linkedinclonebackend/unit/controllers/ConnectionControllerTest.java

```diff
@@ -0,0 +1,153 @@
+package org.autopotato.linkedinclonebackend.unit.controllers;
+
+import static org.autopotato.linkedinclonebackend.controllers.ConnectionController.NewConnectionRequestDTO;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.Collections;
+import org.autopotato.linkedinclonebackend.controllers.ConnectionController;
+import org.autopotato.linkedinclonebackend.model.ConnectionRequest;
+import org.autopotato.linkedinclonebackend.model.Person;
+import org.autopotato.linkedinclonebackend.services.ConnectionRequestService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import org.springframework.test.web.servlet.MockMvc;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+class ConnectionControllerTest {
+    @InjectMocks
+    ConnectionController connectionController;
+
+    @Mock
+    ConnectionRequestService connectionRequestService;
+
+    NewConnectionRequestDTO newConnectionRequestDTO;
+
+    @Autowired
+    MockMvc mockMvc;
+
+    private ObjectMapper jsonMapper;
+
+    public final NewConnectionRequestDTO[] conenctionOptions() {
```

Comment:
There is a typo here. Also, I fail to find where this is used.

Reply:
oh, I will handle that.. stand by

Reply:
done